Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserEntityInspectorWidget.h

```diff
@@ -26,11 +33,32 @@ namespace AzToolsFramework
         public:
             explicit AssetBrowserEntityInspectorWidget(QWidget *parent = nullptr);
             ~AssetBrowserEntityInspectorWidget();
-        private:
+
+            void InitializeDatabase();
+
+            //////////////////////////////////////////////////////////////////////////
+            // AssetBrowserPreviewRequestBus
+            //////////////////////////////////////////////////////////////////////////
             void PreviewAsset(const AzToolsFramework::AssetBrowser::AssetBrowserEntry* selectedEntry) override;
+            void ClearPreview() override;
 
+            void PopulateSourceDependencies(const SourceAssetBrowserEntry* sourceEntry, AZStd::vector<const ProductAssetBrowserEntry*> productList);
+            bool PopulateProductDependencies(const ProductAssetBrowserEntry* productEntry);
+            void CreateSourceDependencyTree(AZStd::set<AZ::Uuid> sourceUuids, bool isOutgoing);
+            void CreateProductDependencyTree(AZStd::set<AZ::Data::AssetId> dependencyUuids, bool isOutgoing);
+            void AddAssetBrowserEntryToTree(const AssetBrowserEntry* entry, QTreeWidgetItem* headerItem);
```

Comment:
A bunch of new public non-override APIs were introduced here. Would be good to add some comments.

Also, it looks like they can be private functions.

Reply:
Made these functions private and added comments to give more context